### PR TITLE
89169632 more feedback

### PR DESCRIPTION
**Pull request by narusemotoki**
Follow Gengo web application.

### CODE REVIEW

**Comment by hayate** [comment]
LGTM
but test failing now.

File: libs/Gengo/Client.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ protected function __construct()
         if (!$useragent)
         {
             // set default user agent string
-            $useragent = 'Gengo PHP Library; Version 2.1; http://gengo.com/';
+            $useragent = 'Gengo PHP Library; Version 2.1test; http://gengo.com/';
```
Comment (seikoudoku2000) [inline libs/Gengo/Client.php]: what is this for?

**Comment by seikoudoku2000** [comment]
LGTM

File: libs/Gengo/Client.php (hunk 1 above)
Comment (narusemotoki) [inline libs/Gengo/Client.php]: Thank you I forgot to revert this change.
